src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,247 @@
+
+Specific MOTION fields MUST be present:
+
+{{ MACROS___make_sidecar_table("motion.motionRequired") }}
+
+| **Key name**       | **Requirement level** | **Data type** | **Description**                                                                                                     |
+| ------------------ | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------- |
+| SamplingFrequency  | REQUIRED              | number        | Nominal sampling rate of the tracking system in Hz.                                                                 |
+| TrackingSystemName | REQUIRED              | string        | Name of the tracking system. The value must match the `tracksys` label of the corresponding *_motion.tsv file name. |
+
+Motion specific fields SHOULD be present:
+
+{{ MACROS___make_sidecar_table("motion.motionMoreRecommended") }}
+
+| **Key name**               | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| -------------------------- | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| MissingValues              | RECOMMENDED           | string        | How missing values are represented in the given tracking system, for example, "NaN", "0".                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Any recording device can provide a `MissingValue` when it is self-aware of it. This is I think most common when the device sends time stamps as well. For MoCap systems, it is quite common as multiple trackers can be used. I think brain recording devices are not so prone to error, but this can be relevant for example for eye-tracking.